Sources/Models/ProfileCertificatesResponse.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  ProfileCertificatesResponse.swift
+//  AppStoreConnect-Swift-SDK
+//
+//  Created by Kristof Van Landschoot on 03/13/20.
```

Comment:
```suggestion
//  Created by Kristof Van Landschoot on 03/14/20.
```

Using the date to trigger a new CI build. Bitrise seems to be a bit stuck 😅